Python 3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.


== RESTART: C:/Users/Я/AppData/Local/Programs/Python/Python311/USMU_MM_125.py ==
USMU_MM_1.2.5
a=20
b-30
c=40
Traceback (most recent call last):
  File "C:/Users/Я/AppData/Local/Programs/Python/Python311/USMU_MM_125.py", line 10, in <module>
    f=e*d/sin(a*b*c/i)
NameError: name 'sin' is not defined. Did you mean: 'bin'?
>>> 
== RESTART: C:/Users/Я/AppData/Local/Programs/Python/Python311/USMU_MM_125.py ==
USMU_MM_1.2.5
a=10
b=20
c=30
1 -0.05281724506350654 740.9142230920022
9 -0.47535520557155886 -4718.874431860118
17 -0.8978931660796111 -6099.308569856293
25 -1.3204311265876634 -8379.742302647901
33 -1.7429690870957157 27229.469666598125
41 -2.165507047603768 -13435.696880693897
49 -2.5880450081118203 -212515.9661027094
57 -3.0105829686198726 18067.040628097966
65 -3.433120929127925 22087.80815485001
73 -3.8556588896359774 -47356.95326834376
81 -4.278196850144029 26470.26151117261
89 -4.700734810652082 28438.923965474314
97 -5.123272771160134 37108.79979877673
105 -5.545810731668187 -59436.67392555412
113 -5.968348692176239 -117486.6105960414
121 -6.390886652684292 61080.24002094621
129 -6.8134246131923435 -71131.24837799408
137 -7.235962573700395 233928.2929671511
145 -7.658500534208448 89583.632831206
153 -8.0810384947165 -48557.59348261575
161 -8.503576455224552 121852.14256195737
169 -8.926114415732606 66057.85677190208
177 -9.348652376240658 -91579.26176771903
185 -9.77119033674871 -68950.87125453782
193 -10.193728297256762 189943.91263970258
201 -10.616266257764813 63698.602074881666
209 -11.038804218272867 157576.41895345316
217 -11.461342178780919 -117603.71611139778
End

